### PR TITLE
DROOLS-2711 : [IE] Package dropdown is broken in new asset dialogs

### PR DESCRIPTION
**Pull request by Rikkola**
https://issues.jboss.org/browse/DROOLS-2711

### CODE REVIEW

**Comment by Rikkola** [comment]
@barboras7 Hi, I could use a review. Thank you.

**Comment by Rikkola** [comment]
For some reason there is a missing tomcat dependency. 
jenkins please retest this.

**Comment by mareknovotny** [comment]
is this updated to the same state as https://github.com/kiegroup/kie-wb-common/pull/1960 ?

**Comment by Rikkola** [comment]
@mareknovotny Yep this was the one I first updated and then cherry-picked to 7.7.x

**Comment by barboras7** [comment]
Jenkins execute full downstream build

**Comment by Rikkola** [comment]
Still the same issue and I do not understand why. Works fine locally.
"[ERROR] Failed to execute goal on project kie-tomcat-integration: Could not resolve dependencies for project org.kie:kie-tomcat-integration:jar:7.9.0-SNAPSHOT"

jenkins please retest this.

**Comment by jhrcek** [comment]
@Rikkola FYI the [PR to fix that error](https://github.com/kiegroup/droolsjbpm-build-bootstrap/pull/773) has been merged

**Comment by barboras7** [comment]
Jenkins execute full downstream build

File: kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/handlers/PackageListBox.java [hunk 1]
```diff
@@ -68,99 +65,100 @@ public void setUp(final boolean includeDefaultPackage) {
...
+    private void showListOfPackages(final boolean includeDefaultPackage,
+                                    final Command packagesLoadedCommand) {
         final Module activeModule = projectContext.getActiveModule().orElse(null);
         if (activeModule == null) {
             return;
+        } else {
+            moduleService.call((Set<Package> pkgs) -> {
+                moduleService.call((Package activePackage) -> {
+                    //Sort by caption
+                    final List<Package> sortedPackages = getPackages(includeDefaultPackage,
```
Comment (jomarko) [inline kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/handlers/PackageListBox.java]: Shouldn't we then call name the method `sortPackages`?

Comment (mareknovotny) [reply kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/handlers/PackageListBox.java]: Isn't  `getSortedPackages` method name more suitable?

Comment (Rikkola) [reply kie-wb-common-widgets/kie-wb-common-ui/src/main/java/org/kie/workbench/common/widgets/client/handlers/PackageListBox.java]: Used `getSortedPackages` both are OK I guess.

**Pull request merged**